frontend/packages/dev-console/integration-tests/features/Knative/connector-Event-Sources-ODC-4230.feature

```diff
@@ -0,0 +1,89 @@
+Feature: CamelK Connector Event Sources
+    User should be able to install Connector Event Sources
+
+
+#Scenarios of installing these operator has been included in operators.feature file
+Background: 
+    Given user has installed the OpenShift Serverless Operator
+    And user has created knative serving
+    And user has create knative eventing
+    And user has installed Knative Apache Camel Operator
+    And user is at Developer perspective
+
+
+@regression
+Scenario: CamelK Connector Event Source cards
+    Given user is at Developer perspective
+    When user goes to +Add page
+    And user clicks on the Event Sources card
+    Then user will see the AWS 2 Simple Queue Service card
```

Comment:
Will the cards are the same you mentioned or may change

Reply:
No. As of now, there won't be any change.